lhotse/dataset/speech_recognition.py

```diff
@@ -132,19 +135,37 @@ def __getitem__(self, cuts: CutSet) -> Dict[str, Union[torch.Tensor, List[str]]]
         segments = torch.stack(list(supervision_intervals.values()), dim=1)
         for tnfm in self.input_transforms:
             inputs = tnfm(inputs, supervision_segments=segments)
-
-        batch = {
-            "inputs": inputs,
-            "supervisions": default_collate(
-                [
-                    {
-                        "text": supervision.text,
```

Comment:
maybe we can just add `”language”: supervision.language,` in the line below and always return it to get rid of the extra option and code duplication.